rsmtool/configuration_parser.py

```diff
@@ -1058,7 +1050,18 @@ def validate_config(cls, config, context="rsmtool"):
                     group: new_config["min_n_per_group"] for group in new_config["subgroups"]
                 }
 
-        # 15. Clean up config dict to keep only context-specific fields
+        # 15. Check that if the user enables logging to W&B, they also
+        # specified wandb project and entity.
+        if new_config["use_wandb"]:
```

Comment:
While I don't mind having an extra key to make the use of W&B explicit, can't we just infer whether W&B is to be used or not based on the presence of `"wandb_project"` and `"wandb_entity"`?

Reply:
Yeah, that's true. Could support faster debugging/development.